daemon/Makefile

```diff
@@ -30,6 +32,7 @@ check-bindata:
 
 go-bindata: $(shell find ../bpf)
 	go-bindata -prefix ../ \
+	-mode 0640 \
```

Comment:
the permissions stored in git don't overwrite the OS permissions, at least on my case, for example I have:
```
$ ls -l
total 292
-rw-------. 1 aanm aanm   935 mar 20 00:15 bpf_features.h
-rw-------. 1 aanm aanm  5637 mar 21 22:21 bpf_lb.c
...
```